examples/bri-3/src/bri/identity/bpiSubjects/api/subjects.controller.spec.ts

```diff
@@ -55,31 +140,58 @@ describe('SubjectController', () => {
     });
   });
 
-  describe('getBpiSubjectById', () => {
+  describe('updateBpiSubject', () => {
```

Comment:
do we need tests with empty name (description and pk)?

Reply:
Laters, once we introduce input DTO validation. Keep in mind that we are still in the early phase and most of the code is merged so that others can follow with their implementations.